source/numerics.tex

```diff
@@ -6752,12 +6752,6 @@
 matrix class nor a field class.
 However, it is a very useful building block for designing such classes.}
 
-\pnum
-An implementation is permitted to qualify any of the functions declared in
-\tcode{<valarray>}
-as
-\tcode{inline}.
-
```

Comment:
Ah ok, was just checking because this edit doesn't match the commit description.